src/AcmTable/AcmTable.tsx

```diff
@@ -192,8 +193,10 @@ export interface AcmTableProps<T> {
     groupSummaryFn?: (items: T[]) => IRow
     tableActions?: IAcmTableAction[]
     rowActions?: IAcmRowAction<T>[]
+    rowActionResolver?: IActionsResolver
```

Comment:
```suggestion
    rowActionResolver?: (item: T) => IActions
```
You would then need to wrap this function before passing to the `Table` in order to resolve the item to pass, but I think it's more in line with the rest of our `AcmTable` interface and more powerful.

Reply:
For the resolver, currently we pass a resolver function to the Patternfly `actionResolver` that conditionally renders actions in the menu as enabled or disabled based on the user's access to resources and namespaces. It also renders action titles such as "Enable"/"Disable" based on the policy's current specification and renders a separator.

Sorry, I'm not following how this change is different/more powerful than using the `IActionsResolver`. This change looks nearly identical to the Patternfly definition: 
```
export type IActionsResolver = (rowData: IRowData, extraData: IExtraData) => (IAction | ISeparator)[];
```
https://github.com/patternfly/patternfly-react/blob/7ecba96a191798d01c7f9209a14a04f4a658d7bb/packages/react-table/src/components/Table/TableTypes.ts#L151